src/ir/item_kind.rs

```diff
@@ -22,6 +24,19 @@ pub enum ItemKind {
     Var(Var),
 }
 
+impl fmt::Display for ItemKind {
```

Comment:
I don't think we should implement `Display` for `ItemKind`. I think this would be better as a method `ItemKind::kind_name(&self) -> &'static str`.